### ISSUE TITLE
Sorry to bother you, but a high loss value occurs during training！

### ISSUE THREAD

**Issue by JioJe** [kohya-ss/musubi-tuner#496]
(venv) D:\musubi-train\1>accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 wan_train_network.py --task t2v-A14B --dataset_config D:\musubi-tuner\train\test\test.toml --sdpa --mixed_precision fp16 --optimizer_type AdamW8bit --logging_dir ./logs --log_with tensorboard --max_data_loader_n_workers 2 --lr_scheduler constant --lr_warmup_steps 0 --network_module networks.lora_wan --learning_rate 1e-4 --network_dim 32 --max_train_epochs 10 --save_every_n_epochs 1 --seed 42 --output_dir D:\musubi-train\musubi-tuner\output --output_name wan22_lora --persistent_data_loader_workers --gradient_checkpointing --gradient_accumulation_steps 1 --fp8_base --fp8_scaled --dit D:\Comfyui_Video\ComfyUI_windows_portable\ComfyUI\models\diffusion_models\wan2.2\T2V\wan2.2_t2v_high_noise_14B_fp16.safetensors --min_timestep 875 --preserve_distribution_shape --blocks_to_swap 36
Trying to import sageattention
Failed to import sageattention
INFO:musubi_tuner.wan.modules.model:Detected DiT dtype: torch.float16
INFO:musubi_tuner.wan_train_network:Converted timestep_boundary to 0 to 1 range: 0.875
INFO:musubi_tuner.hv_train_network:Load dataset config from D:\musubi-tuner\train\test\test.toml
INFO:musubi_tuner.dataset.image_video_dataset:glob videos in D:\musubi-tuner\train\test
INFO:musubi_tuner.dataset.image_video_dataset:found 1 videos
INFO:musubi_tuner.dataset.config_utils:[Dataset 0]
  is_image_dataset: False
  resolution: (416, 240)
  batch_size: 1
  num_repeats: 1
  caption_extension: ".txt"
  enable_bucket: True
  bucket_no_upscale: False
  cache_directory: "D:\musubi-tuner\train\test\cache"
  debug_dataset: False
    video_directory: "D:\musubi-tuner\train\test"
    video_jsonl_file: "None"
    control_directory: "None"
    target_frames: (1, 29, 57, 85)
    frame_extraction: uniform
    frame_stride: 1
    frame_sample: 3
    max_frames: 129
    source_fps: None
    fp_latent_window_size: 9

INFO:musubi_tuner.dataset.image_video_dataset:bucket: (240, 416, 1), count: 3
INFO:musubi_tuner.dataset.image_video_dataset:bucket: (240, 416, 29), count: 3
INFO:musubi_tuner.dataset.image_video_dataset:bucket: (240, 416, 57), count: 3
INFO:musubi_tuner.dataset.image_video_dataset:total batches: 9
INFO:musubi_tuner.hv_train_network:preparing accelerator
accelerator device: cuda
INFO:musubi_tuner.hv_train_network:DiT precision: torch.float16, weight precision: None
INFO:musubi_tuner.hv_train_network:Loading DiT model from D:\Comfyui_Video\ComfyUI_windows_portable\ComfyUI\models\diffusion_models\wan2.2\T2V\wan2.2_t2v_high_noise_14B_fp16.safetensors
INFO:musubi_tuner.wan.modules.model:Creating WanModel. I2V: False, FLF2V: False, V2.2: True, device: cuda, loading_device: cpu, fp8_scaled: True
INFO:musubi_tuner.wan.modules.model:Loading DiT model from D:\Comfyui_Video\ComfyUI_windows_portable\ComfyUI\models\diffusion_models\wan2.2\T2V\wan2.2_t2v_high_noise_14B_fp16.safetensors, device=cpu
INFO:musubi_tuner.utils.lora_utils:Loading model files: ['D:\\Comfyui_Video\\ComfyUI_windows_portable\\ComfyUI\\models\\diffusion_models\\wan2.2\\T2V\\wan2.2_t2v_high_noise_14B_fp16.safetensors']
INFO:musubi_tuner.utils.lora_utils:Loading state dict with FP8 optimization. Dtype of weight: None, hook enabled: False
Loading wan2.2_t2v_high_noise_14B_fp16.safetensors: 100%|█████████████████████████| 1095/1095 [00:34<00:00, 31.89key/s]
INFO:musubi_tuner.modules.fp8_optimization_utils:Number of optimized Linear layers: 400
INFO:musubi_tuner.modules.fp8_optimization_utils:Number of monkey-patched Linear layers: 400
INFO:musubi_tuner.wan.modules.model:Loaded DiT model from D:\Comfyui_Video\ComfyUI_windows_portable\ComfyUI\models\diffusion_models\wan2.2\T2V\wan2.2_t2v_high_noise_14B_fp16.safetensors, info=<All keys matched successfully>
INFO:musubi_tuner.hv_train_network:enable swap 36 blocks to CPU from device: cuda
WanModel: Block swap enabled. Swapping 36 blocks out of 40 blocks. Supports backward: True
import network module: networks.lora_wan
INFO:musubi_tuner.networks.lora:create LoRA network. base dim (rank): 32, alpha: 1
INFO:musubi_tuner.networks.lora:neuron dropout: p=None, rank dropout: p=None, module dropout: p=None
INFO:musubi_tuner.networks.lora:create LoRA for U-Net/DiT: 400 modules.
INFO:musubi_tuner.networks.lora:enable LoRA for U-Net: 400 modules
WanModel: Gradient checkpointing enabled.
prepare optimizer, data loader etc.
INFO:musubi_tuner.hv_train_network:use 8-bit AdamW optimizer | {}
override steps. steps for 10 epochs is / 指定エポックまでのステップ数: 90
running training / 学習開始
  num train items / 学習画像、動画数: 9
  num batches per epoch / 1epochのバッチ数: 9
  num epochs / epoch数: 10
  batch size per device / バッチサイズ: 1
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 90
INFO:musubi_tuner.hv_train_network:set DiT model name for metadata: D:\Comfyui_Video\ComfyUI_windows_portable\ComfyUI\models\diffusion_models\wan2.2\T2V\wan2.2_t2v_high_noise_14B_fp16.safetensors
steps:   0%|                                                                                    | 0/90 [00:00<?, ?it/s]INFO:musubi_tuner.hv_train_network:DiT dtype: torch.float16, device: cuda:0

epoch 1/10
Trying to import sageattention
Trying to import sageattention
Failed to import sageattention
Failed to import sageattention
INFO:musubi_tuner.dataset.image_video_dataset:epoch is incremented. current_epoch: 0, epoch: 1
INFO:musubi_tuner.dataset.image_video_dataset:epoch is incremented. current_epoch: 0, epoch: 1
steps:   8%|████▋                                                        | 7/90 [00:48<09:37,  6.95s/it, avr_loss=1.26]

**Comment by kohya-ss** [comment]
You are training a high noise model with timesteps between 875 and 1000. If the timesteps are close to 1000, the loss value will be large. So, if the training is going well, this loss value is normal.

**Comment by JioJe** [comment]
> You are training a high noise model with timesteps between 875 and 1000. If the timesteps are close to 1000, the loss value will be large. So, if the training is going well, this loss value is normal.

thanks！have a nice day!
